packages/image_picker/image_picker_platform_interface/test/new_method_channel_image_picker_test.dart

```diff
@@ -462,6 +462,22 @@ void main() {
         expect(response.file!.path, '/example/path');
       });
 
+      test('retrieveLostData get success response with pathList', () async {
```

Comment:
Can you also add tests for when the `retrieveLostData` fails and it should fall back on the single image?